docker-compose.yml

```diff
@@ -12,6 +12,10 @@ services:
       retries: 10
     volumes:
       - ./src/main/resources/buildings.sql:/docker-entrypoint-initdb.d/buildings.sql
+      - ./src/main/resources/roads.sql:/docker-entrypoint-initdb.d/roads.sql
+      - ./src/main/resources/rivers.sql:/docker-entrypoint-initdb.d/rivers.sql
+      - ./src/main/resources/forests.sql:/docker-entrypoint-initdb.d/forests.sql
+      - ./src/main/resources/fields.sql:/docker-entrypoint-initdb.d/fields.sql
```

Comment:
I think it will be good to add info to help MD to copy this files to proper dirctory

Reply:
And this files shoudl be added to .gitignore I think